topic: "/visualization_marker_array"
map_topic: "/map"
needle: "buildings_osm"
enable_tf: 1
type : "polygon"
map_frame: "map"
size_x: 100.0
size_y: 100.0

TF:
  enable_tf: 1
  origin_frame: "world"
  output_frame: "map"
